qa/automation/CareerPortal.Tests.Scenarios/2. Extended example/Test/CVforNotOpenPosition.feature

```diff
@@ -0,0 +1,122 @@
+Background: User went to the "Send us your CV" form at the down of the page 
+
+Feature: Send CV for not open job position  
+
+@Happy_pass
+Scenario: Send CV happy pass
+Given User entered valid Name
```

Comment:
Here and in other Scenarios you need to use real values, instead of "valid name", "valid Email", etc.

Reply:
Done! Thanks!